### PR TITLE
[QC-1008] shm-metadata-msg-size parameter mapped

### PR DESCRIPTION
**Pull request by justonedev1**
- parameter mapped
- added simple workflow to test the parameter values

### CODE REVIEW

**Comment by justonedev1** [comment]
This task was issued by @knopers8

File: Framework/TestWorkflows/src/o2FairMQHeaderSizeTest.cxx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
```
Comment (knopers8) [inline Framework/TestWorkflows/src/o2FairMQHeaderSizeTest.cxx:11]: Could you document in a few sentences what is this executable for?

Comment (justonedev1) [reply Framework/TestWorkflows/src/o2FairMQHeaderSizeTest.cxx:11]: done

**Review by knopers8 — DISMISSED** [review]
Thank you!
@ktf Are you OK with this?

**Comment by knopers8** [comment]
> Thank you! @ktf Are you OK with this?

ping @ktf

**Comment by ktf** [comment]
I think you might need the extra parameter in a few more places, actually. At least that's the case for all the others "shm-" ones.

**Review by ktf — CHANGES_REQUESTED** [review]
I think you need the parameter to be propagated in a few more places, like the others.

**Comment by justonedev1** [comment]
Is this better @ktf ?

File: Framework/Core/test/test_FrameworkDataFlowToDDS.cxx [hunk 2]
```diff
@@ -379,6 +379,8 @@ TEST_CASE("TestDDS")
    </declcollection>
 </topology>
 )EXPECTED";
+  LOG(info) << ss.str();
+  LOG(info) << expected;
```
Comment (ktf) [inline Framework/Core/test/test_FrameworkDataFlowToDDS.cxx]:
```suggestion
  LOG(info) << ss.str();
  LOG(info) << expected;
```

no printouts in the tests, please.

**Comment by justonedev1** [comment]
@ktf I removed the logging lines and rebased it on current dev branch so CI works. Is there anything else I need to change?

**Comment by knopers8** [comment]
@ktf ping

**Comment by justonedev1** [comment]
@ktf ping

**Pull request merged**